cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-rosa-sts-explained.adoc

```diff
@@ -0,0 +1,147 @@
+:_content-type: ASSEMBLY
+[id="cloud-experts-rosa-sts-explained"]
+= Tutorial: ROSA with AWS STS explained
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-rosa-sts-explained
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-10-26
+
+This tutorial outlines the two options for allowing {product-title} (ROSA) to interact with resources in a user's Amazon Web Service (AWS) account. It details the components and processes that ROSA with Security Token Service (STS) uses to obtain the necessary credentials. It also reviews why ROSA with STS is the more secure, preferred method. 
+
+This tutorial will:
+
+* Enumerate the two main deployment options:
```

Comment:
One suggestion: 
Since we have ROSA with HCP, maybe add a not that this content does not yet cover HCP and change this bullet to. "* Enumerate two of the deployment options:"